coil-sample/build.gradle.kts

```diff
@@ -42,6 +42,7 @@ android {
 
 dependencies {
     implementation(project(":coil-default"))
+    implementation(project(":coil-svg"))
```

Comment:
Thanks for adding this, however I'd like to leave the `coil-svg` module out of the sample app for the moment. Going to work on a separate PR to have new screens for the `gif` and `svg` modules.